.github/workflows/cicd-website.yml

```diff
@@ -0,0 +1,92 @@
+name: CI/CD Website branch
+
+on:
+  push:
+    branches:
+      - website
+  pull_request:
+    branches:
+      - website
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Set up Node.js version
+        uses: actions/setup-node@v3.2.0
+        with:
+          node-version: "16.13.x"
+
+      - name: yarn install
+        run: |
+          npm install -g yarn
+          yarn install --frozen-lockfile
+
+      - name: run linters
+        run: |
+          yarn lint:css
+          yarn lint:package
+          yarn lint:js
+          yarn lint:prettier
+
+      - name: build library
+        run: yarn build:library
+
+      - name: typecheck storybook
+        run: yarn typecheck:storybook
+
+      - name: build storybook
+        run: yarn build:storybook
+
+  publish-npm:
+    runs-on: ubuntu-latest
+    needs: build
+    if: github.ref == 'refs/heads/website'
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GH_ADMIN_TOKEN }}
+
+      - name: Set up Node.js version
+        uses: actions/setup-node@v3.2.0
+        with:
+          node-version: "16.13.x"
+
+      - name: yarn install
+        run: |
+          npm install -g yarn
+          yarn install --frozen-lockfile
+
+      - name: build library
+        run: yarn build
+
+      - name: lerna version
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_ADMIN_TOKEN }}
+          GIT_AUTHOR_EMAIL: ${{ secrets.GIT_AUTHOR_EMAIL }}
+          GIT_AUTHOR_NAME: "Den Haag website"
+          GIT_COMMITTER_EMAIL: ${{ secrets.GIT_COMMITTER_EMAIL }}
+          GIT_COMMITTER_NAME: "Den Haag website"
+        run: |
+          git push --set-upstream origin HEAD
+          yarn release -- --yes
+
+      - name: lerna publish
+        with:
+          node-version: "16.13.x"
+          cache: "npm"
+          registry-url: "https://npm.pkg.github.com/"
+          scope: "@gemeente-denhaag"
+        env:
+          NPM_ACCESS_TOKEN: ${{ secrets.NPM_ACCESS_TOKEN }}
+        run: |
+          npm config set "//npm.pkg.github.com/:_authToken" "${NPM_ACCESS_TOKEN}"
+          npm config set access public
+          npm run publish -- --no-verify-access --yes
+          npm config delete "//npm.pkg.github.com/:_authToken"
```

Comment:
We might need to configure lerna or all packages to use a slightly different type of versioning. Ideally we avoid that `alpha.101` means one thing in npm and another thing in the github registry.

If we need to update all the `package.json` files to use `website.101` instead, it will lead to major merge conflicts since those lines change every release.

Reply:
Good point added `--preid denhaagnl`